build.sh

```diff
@@ -67,6 +68,7 @@ sqlite_year="2016"
 sqlite_ver="3.14.2"
 openslide_ver="3.4.1"
 openslidejava_ver="0.12.2"
+libzip_ver="1.1.2"
```

Comment:
Please be consistent about where in the list you put libzip.  I think it makes sense to put it immediately after zlib.